qa/tasks/cephfs/kernel_mount.py

```diff
@@ -107,6 +108,8 @@ def _get_mount_cmd(self, mntopts):
             opts += ",rbytes"
         else:
             opts += ",norbytes"
+        if self.snapdirname:
```

Comment:
Maybe we can just skip this if `self.snapdirname == '.snap'` ?

Reply:
So we can replace the check with this `if self.snapdirname != '.snap':
            opts += f',snapdirname={self.snapdirname}'`

Reply:
Looks good.